code/modules/projectiles/projectile/beams.dm

```diff
@@ -46,25 +46,31 @@ In pvp they also have more lasting damages, such as infections, and pain form bu
 
 /obj/item/projectile/beam/weak/pistol_35
 	damage_types = list(BURN = 12)
+	armor_penetration = 5
 	recoil = 1
 
 /obj/item/projectile/beam/weak/light_rifle_257
-	damage_types = list(BURN = 13)
+	damage_types = list(BURN = 18)
+	armor_penetration = 15
 
 /obj/item/projectile/beam/weak/rifle_75
 	damage_types = list(BURN = 22)
+	armor_penetration = 20
 	recoil = 4
 
 /obj/item/projectile/beam/weak/heavy_rifle_408
-	damage_types = list(BURN = 16)
+	damage_types = list(BURN = 20)
```

Comment:
This will be way to op in pve as it deals 80 damage to mobs

Reply:
Huh, that is interesting. I am curious to know where this 20 damage gets converted in 80.

Reply:
Went into a local test to see if this is true. I spawned in a STS with a laser bullet with its damage set to 16 and one set to 20.
My results are 47.6 damage with 16 dmg. 52 with 20 dmg.
The 80 damage conversation cannot be found.

Reply:
1 damage mod with a 1 ap mod with vig of 0
![image](https://user-images.githubusercontent.com/30435998/178080440-59b48feb-bba6-4675-b965-d413d0fdd35b.png)
having basic vig will e-z do 65+
I stated 80 do to thinking with 1.2 pen guns rather then defaults
and stats

Reply:
Note this is with **SERVER NUMBERS**
15 more ap (that gets dobled when in terms of damage) to that with the extra 4 more burns
can with easy deal 80 damage